x/twap/strategy.go

```diff
@@ -0,0 +1,40 @@
+package twap
+
+import (
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/osmosis-labs/osmosis/v13/x/twap/types"
+)
+
+type twapStrategy interface {
```

Comment:
Let's add godocs please for each of:
- `twapStrategy`
- `getTwapToNow`
- `computeTwap`